test/intl_suggestors/utils_test.dart

```diff
@@ -87,33 +86,44 @@ void main() {
 
         final parsedInterpolation = parsedExpression as StringInterpolation;
         expect(parsedInterpolation.isMultiline, isMultiline);
-
         final testResult = IntlMessages('Test')
             .syntax
             .functionDefinition(parsedInterpolation, 'Namespace', "NamePrefix");
         expect(testResult, expectedResult);
       }
 
+      void ignoreSingleLineAndMultiline(
+          String testStr, bool isMultiline, String expectedResult) async {
+        final parsedExpression = await sharedContext.parseExpression(testStr);
+        expect(parsedExpression is StringInterpolation, isTrue);
+        var parsedInterpolation = parsedExpression as StringInterpolation;
+        expect(parsedInterpolation.isMultiline, isMultiline);
+        var data = textFromInterpolation(parsedInterpolation);
+        if (data.isEmpty) {
+          data = expectedResult;
+          expect(testStr, expectedResult);
```

Comment:
Isn't this the same expect as the one below this if statement?

Reply:
Removed unnecessary `expect` function call and updated the test case logic